shell/platform/tizen/tizen_view_elementary.cc

```diff
@@ -313,11 +322,7 @@ TizenGeometry TizenViewElementary::GetGeometry() {
 }
 
 void TizenViewElementary::SetGeometry(TizenGeometry geometry) {
-  EvasObjectResize(image_, geometry.width, geometry.height);
-  evas_object_move(image_, geometry.left, geometry.top);
-  evas_object_image_size_set(image_, geometry.width, geometry.height);
-
-  EvasObjectResize(container_, geometry.width, geometry.height);
+  EvasObjectResizeWithMinMaxHint(container_, geometry.width, geometry.height);
   evas_object_move(container_, geometry.left, geometry.top);
```

Comment:
The list below is the resulting evas_object apis.

evas api calls
[before code]
```
  evas_object_resize(image_, width, height);
  evas_object_size_hint_min_set(image_, width, height);
  evas_object_size_hint_max_set(image_, width, height);

  evas_object_move(image_, geometry.left, geometry.top);
  evas_object_image_size_set(image_, geometry.width, geometry.height);

  evas_object_resize(container_, width, height);
  evas_object_size_hint_min_set(container_, width, height);
  evas_object_size_hint_max_set(container_, width, height);

  evas_object_move(container_, geometry.left, geometry.top);
```

[this pr]
```
 evas_object_size_hint_min_set(container_, width, height);
  evas_object_size_hint_max_set(container_, width, height);

  evas_object_resize(image_, width, height);
  evas_object_size_hint_min_set(image_, width, height);
  evas_object_size_hint_max_set(image_, width, height);

  evas_object_move(self->image_, x, y);

  evas_object_resize(container_, width, height);
  evas_object_size_hint_min_set(container_, width, height);
  evas_object_size_hint_max_set(container_, width, height);
  evas_object_move(container_, geometry.left, geometry.top);
```

In this PR, the min max set of the container is did twice.
And this pr has separated the min max set for image in a separate location.
The callback's evas api call should be handled by SetGeometry.

Reply:
I miscalculated the evas api call of container_ called in this PR. Please ignore this comment